diesel_derives/src/selectable.rs

```diff
@@ -47,7 +47,14 @@ pub fn derive(item: DeriveInput) -> Result<TokenStream> {
         .map(|f| field_column_inst(f, &model))
         .collect::<Result<Vec<_>>>()?;
 
-    let check_function = if let Some(ref backends) = model.check_for_backend {
+    let check_function = if let Some(backends) = if cfg!(debug_assertions) {
+        Some(Cow::Owned(syn::parse::Parser::parse2(
+            syn::punctuated::Punctuated::parse_terminated,
+            quote!(diesel::pg::Pg).into(),
```

Comment:
Can you please benchmark the impact of this on compilation time Operations?

Testing process:
- Disable RA
- Clean build
- Add a new field to a Selectable impl (rust-analyzer disabled)
- Time cargo check
- Change diesel revision to this fork
- Remove new field
- Clean build
- Add new field again
- Time cargo check